components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -80,9 +80,20 @@ impl Manifest {
             PackageIdent::from_str(pkg_ident_str)?
         };
 
+        let tag = match pkg_ident.version {
+            Some(v) => {
+                match pkg_ident.release {
```

Comment:
I wonder if there's a way to do this with a chain of functions, something like: `pkg_ident.release.if_present(|r| format!(...))`.

Basically a way to inline the `match`.

Reply:
for transforming the content of `Some`, there is Option::map() so i think it's doable. I'll check it out.

Reply:
i managed to get rid of the inner match (against release) but couldn't manage to do the same for the outer one w/o involving a string copy.